dev_scripts/dataset_management/modify_dataset_metadata.py

```diff
@@ -0,0 +1,203 @@
+from pprint import pprint
+from copy import deepcopy
+import json
+from itertools import compress
+import os
+
+
+from matminer.datasets.utils import _load_dataset_dict
+
+intro_message = """
+How would you like to interact with the dataset metadata?
+(1) See Current Datasets
+(2) Remove a Dataset
+(3) Add a Dataset
+(s) Save changes
+(q) Quit
+"""
+
+
+def generate_new_dataset(dataset):
```

Comment:
Could you document what this function does and its inputs?

Reply:
Done